README.md

```diff
@@ -67,10 +67,10 @@ parameters:
 	reportUnmatchedIgnoredErrors: false
 	# Ignore phpstan-drupal extension's rules.
 	ignoreErrors:
-		- '#\Drupal calls should be avoided in classes, use dependency injection instead#'
-		- '#Plugin definitions cannot be altered.#'
-		- '#Missing cache backend declaration for performance.#'
-		- '#Plugin manager has cache backend specified but does not declare cache tags.#'
+		- '#^\\Drupal calls should be avoided in classes, use dependency injection instead$#D'
+		- '#^Plugin definitions cannot be altered\.$#D'
+		- '#^Missing cache backend declaration for performance\.#D'
+		- '#^Plugin manager has cache backend specified but does not declare cache tags\.#D'
```

Comment:
Are these not working anymore? These are still used as-is with Upgrade Status https://git.drupalcode.org/project/upgrade_status/-/blob/8.x-3.x/deprecation_testing_template.neon

Reply:
I see, didn't know about `upgrade_status` project, thanks.

However, the `ignoreErrors` messages are regexes, so they are working just by coincidence; I should maybe create a MR in https://git.drupalcode.org/project/upgrade_status...

Reply:
upgrade_status copied from drupal-check, and this README.

See https://phpstan.org/user-guide/ignoring-errors#ignoring-in-configuration-file

While your suggestion improves them, they still work as currently written. I have yet to test them out.